1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,7 +16,7 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+let thingsICanBuy = store.filter(item => item.price <= 1.5).map(item => item.price);
```

Comment:
Your use of filter and array are great. But what would happen if changed the value of money from 1.5 to 2.0? Are you using this variable?